### DISCUSSION TITLE
PIP v2.1 AVAILABLE

### DISCUSSION THREAD

**Discussion by dpuenteramirez** [dpuenteramirez/Semisupervised-learning-and-instance-selection-methods#102] [Announcements]
Since of now the project libraries are available to download from PIP. As it may seem, both are in early stages of release, as the project grow older will be updated with the necessary items to make them look as professional as any other project.

[Instance Selection](https://pypi.org/project/IS-DNX/)
[Semi-Supervised Learning](https://pypi.org/project/SSL-DNX/)

**Comment by alvarag** [comment]
It looks great. You can improve the project with description and things like that

**Reply by dpuenteramirez** [reply]
Both actually have a description, but for some reason only in v.0.1.0, the following do not 😅 I'm working on it

**Reply by dpuenteramirez** [reply]
In v.0.1.3, the lastest as of right now, both have a project description. It had to do with the [setup.cfg](https://github.com/dpr1005/Semisupervised-learning-and-instance-selection-methods/blob/main/setup.cfg) that it was making some odd things.

**Comment by dpuenteramirez** [comment]
PIP v.2.1 is now available, algorithms related to Density Peaks are now available
